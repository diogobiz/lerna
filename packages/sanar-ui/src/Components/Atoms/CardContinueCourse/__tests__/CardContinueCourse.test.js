
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESCardContinueCourse from '../'

it('renders correctly', () => {
    const component = (
        <ESCardContinueCourse>
            <h1>ESCardContinueCourse</h1>
        </ESCardContinueCourse>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
