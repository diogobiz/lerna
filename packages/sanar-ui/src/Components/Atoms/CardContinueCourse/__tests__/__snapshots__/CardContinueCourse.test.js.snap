// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-card-continue-course es-card-continue-course--radius"
>
  <div
    className="es-card-continue-course--texts"
  >
    <div
      className="es-typography text-grey-6 es-typography--caption es-typography--ellipsis"
      data-testid="es-typography"
    />
    <div
      className="es-typography text-grey-8 es-typography--caption es-typography--strong es-typography--ellipsis"
      data-testid="es-typography"
    />
  </div>
  <i
    className="anticon es-eva-icon es-eva-icon--xlarge"
  >
    <i
      data-eva="play-circle"
      data-eva-fill="#fff"
      data-eva-height="18px"
      data-eva-width="18px"
    />
  </i>
</div>
`;
